web/test/helpers/helpers-formation.tsx

```diff
@@ -179,12 +184,12 @@ export type FormationPageHelpers = {
   submitBillingStep: () => Promise<void>;
   submitReviewStep: () => Promise<void>;
   clickSubmitAndGetError: (business: Business) => Promise<void>;
-  stepperClickToBusinessNameStep: () => Promise<void>;
-  stepperClickToNexusBusinessNameStep: () => Promise<void>;
-  stepperClickToBusinessStep: () => Promise<void>;
-  stepperClickToContactsStep: () => Promise<void>;
-  stepperClickToBillingStep: () => Promise<void>;
-  stepperClickToReviewStep: () => Promise<void>;
+  stepperClickToBusinessNameStep: (optionUserEvent?: optionUserEvent) => Promise<void>;
+  stepperClickToNexusBusinessNameStep: (optionUserEvent?: optionUserEvent) => Promise<void>;
+  stepperClickToBusinessStep: (optionUserEvent?: optionUserEvent) => Promise<void>;
+  stepperClickToContactsStep: (optionUserEvent?: optionUserEvent) => Promise<void>;
+  stepperClickToBillingStep: (optionUserEvent?: optionUserEvent) => Promise<void>;
+  stepperClickToReviewStep: (optionUserEvent?: optionUserEvent) => Promise<void>;
```

Comment:
[boulder] I would not name the parameter the same as its type and I'd capitalize the type name